### DISCUSSION TITLE
Supporting the VIM global command

### DISCUSSION THREAD

**Discussion by korgoth** [zed-industries/zed#19198] [Feature Requests]
Is it possible to support the VIM g command? 

Its extremely useful for quick filtering large files and executing actions on the filtered results

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
